Graphics/Implicit/Definitions.hs

```diff
@@ -1,18 +1,17 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
ok, this is a bit insane. :D

Reply:
right :)

easy to fix, though, and if you or somebody always takes a last look at all commits before merging, it'll be hard to miss these.